conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -21,15 +21,14 @@ class Command(BaseCommand):
     Creates bunch of data that's required for new developer setup.
     Mostly django CMS pages.
     """
+    @transaction.atomic
     def handle(self, *args, **options):
 
-        Conference.objects.get_or_create(
-            code=settings.CONFERENCE_CONFERENCE,
-            name=settings.CONFERENCE_CONFERENCE,
-        )
+        homepage = create_homepage_in_cms()
```

Comment:
I'd keep the Conference.get_or_create there even though it's redundant – so it serves a documentation that the Conference instance is required (when somebody reads the script to learn how to set up initial data)